docs/guides/setup/NCI/account.rst

```diff
@@ -69,19 +69,12 @@ Use `MyNCI <https://my.nci.org.au/>`_ to view and manage your project membership
      - Geoscience Australia Sentinel-2 Analysis Ready Data (Collection 3)
      
    * - jw04_
-     - Geoscience Australia Landsat Derivatives (Collection 3), e.g. WO, FC
+     - Geoscience Australia Landsat Derivatives (Collection 3), e.g. WO, FC, GeoMAD, Intertidal
 
    * - fk4_
      - Legacy Geoscience Australia Landsat Derivatives (Collection 2), e.g. NIDEM, ITEM, HLTC
```

Comment:
> WO, FC, GeoMAD, Intertidal

> NIDEM, ITEM, HLTC

@robbibt Are these abbreviations used within the NCI? Otherwise, I think the reader may not know what these mean. Could we spell out the products by the full names that are used in NCI?

Reply:
I've updated the text for `jw04` - for `fk4` I'm kind of happy for them to be obscure and have users not recognise them unless they're specifically looking for them. We don't really want to encourage users to access those older products unless they really have to (hopefully soon we'll have published new version of all of them)

Reply:
That's looking better. Would it be ok if I add a link to each of these and spell out the name in more detail. I want to make it as clear as possible for the reader. (They can click the link to see the product page to decide if they want to use/not use the product if it's a legacy product.)

Reply:
No problem adding extra links in there, that sounds great. Just note that that wasn't an exhaustive list of data stored in `jw04` - just a few example products (there's a lot more in there I think).

Spelling out the other product names is just a little sensitive, as they represent older (but actually more recognisable) acronyms for deprecated products. e.g. NIDEM was known as NIDEM for many years until it was temporarily renamed to "DEA Intertidal Elevation", and then that product was entirely deprecated in favour of "DEA Intertidal". Same for ITEM: it was known as ITEM, then temporarily renamed to "DEA Intertidal Extents", which will hopefully soon be deprecated in favour of "DEA Intertidal" too. So including the full "DEA Intertidal Elevation" and "DEA Intertidal Extents" here adds a risk that users will click on them thinking that they are our latest products, even though they are either deprecated or soon to be replaced.

Would something this be OK, combining the temporary names with the original acronyms, and clarifying that some of them are deprecated? "DEA Intertidal Elevation (NIDEM; deprecated), DEA Intertidal Extents (ITEM), DEA High and Low Tide Composites (HLTC)". Although honestly I think that might be a bit more complex than it needs to be - just adding a hyperlink to the NIDEM, ITEM, HLTC acronyms could be fine I think.

Reply:
Here's the data on `jw04`: 
![image](https://github.com/user-attachments/assets/d4861819-fb55-4a9d-8acd-0db9f0841667)

Reply:
Hey @benji-glitsos-ga , any chance we could deal with renaming those project acronyms in a different PR? Keen to merge this in so we can add NCI access to Intertidal 🙂

Reply:
Hi @robbibt, sorry I forgot to reply to your comment. Here's a possible solution - we could use the 'short' names of the products, plus a link:

```rst
- Geoscience Australia Landsat Derivatives (Collection 3), e.g. `Water Observations </data/product/dea-water-observations-landsat/>`_, `Fractional Cover </data/product/dea-fractional-cover-landsat/>`_, `GeoMAD </data/product/dea-geometric-median-and-median-absolute-deviation-landsat/>`_, and `Intertidal </data/product/dea-intertidal/>`_.
```

Reply:
@benji-glitsos-ga OK, please have another look now